packages/amplication-client/src/Project/ProjectSideBarFooter.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import InnerTabLink from "../Layout/InnerTabLink";
+import "./ProjectSideBarFooter.scss";
+
+const CLASS_NAME = "sidebar-footer ";
+
+const ProjectSideBarFooter = () => {
```

Comment:
I think we should not use "footer" in the name as this component is not directly related to the position where it is delayed. 
Also, it is not related to the project - instead is it related to the workspace. 

I would call it - WorkspaceSettingsNavigation